website/server/models/group.js

```diff
@@ -867,6 +868,10 @@ schema.methods.leave = async function leaveGroup (user, keep = 'keep-all') {
 
   // If user is the last one in group and group is private, delete it
   if (group.memberCount <= 1 && group.privacy === 'private') {
+    let plan = group.purchased.plan;
+    if (plan && plan.customerId && !plan.dateTerminated) {
+      throw new NotAuthorized(shared.i18n.t('cannotDeleteActiveAccount'));
```

Comment:
I think it deserves a new string specific for groups. Also, just thought about the case when the leader deleted its account, what happens to the group plan?